stdlib/src/array.rs

```diff
@@ -1258,9 +1258,11 @@ mod array {
 
     impl Iterable for PyArray {
         fn iter(zelf: PyRef<Self>, vm: &VirtualMachine) -> PyResult {
+            let zelf_ = zelf.clone();
             Ok(PyArrayIter {
                 position: AtomicUsize::new(0),
                 array: zelf,
```

Comment:
```suggestion
                array: zelf.clone(),
```
change here to allow to use zelf later.